README.md

```diff
@@ -23,7 +23,7 @@ ember install ember-frost-sort
 ```
 
 ## API
-Coming soon
+Coming soon...
```

Comment:
When you want three dots please use `…` instead of three periods.On a Mac you can type this character by doing `Alt + ;`, on Windows it is `Alt + 0133` on the Numpad.